framework/interface/urls.py

```diff
@@ -34,6 +34,7 @@ def get_handlers():
         tornado.web.url(r'/api/targets/([0-9]+)/transactions/hrt/?([0-9]+)?/?$', api_handlers.TransactionHrtHandler, name='transactions_hrt_api_url'),
         tornado.web.url(r'/api/targets/([0-9]+)/poutput/?' + plugin_group_re + '/?' + plugin_type_re + '/?' + plugin_code_re + '/?$', api_handlers.PluginOutputHandler, name='poutput_api_url'),
         tornado.web.url(r'/api/targets/([0-9]+)/poutput/names/?' + plugin_group_re + '/?' + plugin_type_re + '/?' + plugin_code_re + '/?$', api_handlers.PluginNameOutput, name='plugin_name_api_url'),
+        tornado.web.url(r'/api/targets/([0-9]+)/export/?$', api_handlers.ReportExportHandler, name='report_export_api_url'),
```

Comment:
It might befar-fetched but wouldn't it be more modular to have the API set to `/api/targets/{id}/export/{format}` or `/api/targets/{id}/export/{template}` instead?

It would free the `/api/targets/{id}/export/` API that could later default to a specific template/format, or be used to show a dialog box where the user can specify more options.

What do you think?